ICSharpCode.CodeConverter/CSharp/NodesVisitor.cs

```diff
@@ -1376,11 +1376,16 @@ public override CSharpSyntaxNode VisitBinaryConditionalExpression(VBSyntax.Binar
 
             public override CSharpSyntaxNode VisitTernaryConditionalExpression(VBSyntax.TernaryConditionalExpressionSyntax node)
             {
-                return SyntaxFactory.ConditionalExpression(
+                var expr = SyntaxFactory.ConditionalExpression(
                     (ExpressionSyntax)node.Condition.Accept(TriviaConvertingVisitor),
                     (ExpressionSyntax)node.WhenTrue.Accept(TriviaConvertingVisitor),
                     (ExpressionSyntax)node.WhenFalse.Accept(TriviaConvertingVisitor)
                 );
+
+                if (node.Parent.GetType() == typeof(VBSyntax.InterpolationSyntax))
+                    return SyntaxFactory.ParenthesizedExpression(expr);
```

Comment:
To do this sort of check it's generally preferential to use `node.Parent is VBSyntax.InterpolationSyntax`. This is on the assumption that if a future version of the library had a subclass, that we'd likely also want the same behaviour for that subclass.

In the case of this specific library, there's a special method for these sorts of checks because they're so commonly needed, which provides an abstraction layer over the type system. So you can call something like `node.Parent.IsKind(SyntaxKind.InterpolatedStringExpression)`.